contracts/Automate.sol

```diff
@@ -155,15 +156,36 @@ contract Automate is Gelatofied, Proxied, AutomateStorage, IAutomate {
             success
         );
 
-        emit LogUseGelato1Balance(
-            _oneBalanceParam.sponsor,
+        emit LogUseGelato1Balance(_oneBalanceParam.correlationId);
+    }
+
+    function exec1BalanceSimple(
```

Comment:
I don't really like the `Simple` naming :D

Reply:
Some other suggestions 😄
- exec
- exec1BalanceOptimised
- execBypassModule